src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -398,10 +417,20 @@ private void setState(State state) {
     this.state = state;
   }
 
+  private void setIsFirst(boolean isFirst) {
+    this.isFirst = isFirst;
```

Comment:
Not really critical, but this method doesn't seem necessary since it's just used internally to set a single variable.